README.rst

```diff
@@ -4,6 +4,10 @@
    :target: https://travis-ci.org/nilearn/nilearn
    :alt: Build Status
 
+.. image:: https://ci.appveyor.com/api/projects/status/dn3b752wboo83taw?svg=true
+   :target: https://ci.appveyor.com/project/aabadie/nilearn
```

Comment:
Could you create a nilearn-ci account and change the URL?

The target of the image seems overly complicated, is there a good reason for it? The scikit-learn one is:
https://ci.appveyor.com/api/projects/status/github/scikit-learn/scikit-learn?branch=master&svg=true

so I guess we can use something similar.